packages/main/src/Table.ts

```diff
@@ -464,11 +465,17 @@ class Table extends UI5Element {
 	}
 
 	get styles() {
-		return {
+		const tableStyle = {
 			table: {
 				"grid-template-columns": this._gridTemplateColumns,
 			},
-		};
+		} as any;
+
+		this.headerRow[0].cells.forEach(headerCell => {
+			tableStyle.table[`--h-align-${(headerCell as any)._individualSlot}`] = headerCell.hAlign || TableCellHorizontalAlign.Left;
```

Comment:
Is it really necessary to set a CSS variable for **every** header cell? In the end, we might end up with 20 different CSS variables, even though only 2 cells might have actually set the hAlign property.
Maybe it is possible to only set the variable for header cells that actually changed their values.

Reply:
Now the value for justify-content will always be the name of the css variable but the variable itself is only declared if the headerCell has an valid hAlign attribute.
The jsdoc description of the default value "Start" is technically still correct. But since justify-content is not set if hAlign is not used the default value for justify-content is "normal" in this case which just happen to behaves like "Start".
I guess the jsdoc should then be changed to normal instead?

Reply:
Changed the jsdoc @default to "Normal"